rdflib/extras/infixowl.py

```diff
@@ -278,47 +278,50 @@ def castToQName(x):
                     return ":".join([prefix, localName])
 
                 if len(named) > 1:
-                    prefix = u"( " + u" AND ".join(map(castToQName, named)) + u" )"
+                    prefix = '( ' + ' AND '.join(map(
+                        castToQName, named)) + ' )'
                 else:
                     prefix = manchesterSyntax(named[0], store)
                 if childList:
-                    return (
-                        str(prefix)
-                        + u" THAT "
-                        + u" AND ".join(
-                            [str(manchesterSyntax(x, store)) for x in childList]
-                        )
-                    )
+                    return str(prefix) + ' THAT ' + ' AND '.join(
+                        [str(manchesterSyntax(x, store)) for x in childList])
```

Comment:
Would it still be possible to split 2to3 changes and code formatting in two separate commits? This would improve the clarity for code reviews and possible future code blame.

Reply:
I'll fix the worst of them, ok